src/github.com/stellar/horizon/resource/account.go

```diff
@@ -71,3 +72,16 @@ func (this *Account) Populate(
 	this.Links.Data.PopulateTemplated()
 	return
 }
+
+// GetData returns decoded value for a given key. If the key does
+// not exist, empty slice will be returned.
+func (this *Account) GetData(key string) []byte {
+	data, exists := this.Data[key]
+	if !exists {
+		return nil
+	}
+
+	// We assume that horizon always returns valid base64 encoded string
+	decoded, _ := base64.StdEncoding.DecodeString(data)
```

Comment:
So are you willing to accept this if I added a data value validation in ingester?

I think the version of this method that returns error doesn't really add value for developers since they could achieve the same effect using `encoding/base64`. The purpose of this method is to get decoded value, without having to handle errors. So for example: you can pass it directly as an argument to other function.